panther_lights/src/driver_node.cpp

```diff
@@ -57,6 +58,9 @@ void DriverNode::Initialize()
   frame_timeout_ = this->get_parameter("frame_timeout").as_double();
   num_led_ = this->get_parameter("num_led").as_int();
 
+  std::vector<panther_gpiod::GPIOInfo> gpio_info_storage = {panther_gpiod::GPIOInfo{
+    panther_gpiod::GPIOPin::LED_SBC_SEL, gpiod::line::direction::OUTPUT, true}};
+  gpio_driver_ = std::make_unique<panther_gpiod::GPIODriver>(gpio_info_storage);
   SetPowerPin(false);
```

Comment:
Is it still necessary? The GPIODriver constructor establishes the initial pin value, which in this case is set to INACTIVE.